src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -245,6 +246,9 @@ public void createIndex(String indexDefinition) {
 				list.add(t);
 			}
 			return list;
+		 } catch (UnsupportedEncodingException e) {
+			// This should never happen as every implementation of the java platform is required to support UTF-8.
+			return null;

```

Comment:
Just for discussion: I'm wondering if we should just throw a RuntimeException since we are sure it is never going to happen on java platforms, what are your thoughts @brynh